tests/System.IO.Pipelines.Tests/ReadableBufferFacts.cs

```diff
@@ -354,7 +354,7 @@ private unsafe void TestValue(ref ReadableBuffer readBuffer, ulong value)
             }
         }
 
-        [Fact]
+        [Fact(Skip = "System.TypeLoadException : A value type containing a by-ref instance field, such as Span<T>, cannot be used as the type for a class instance field.")]
```

Comment:
I agree with you. However, I do not have the capacity to work on these right now. This is why I have filed issues to track this work for later or so others can take a shot at it if they can.